### ISSUE TITLE
A few questions

### ISSUE THREAD

**Issue by amuvarma13** [hertz-pj/SNAC-Vocos#7]
Thanks for making this - it's great. We had a few questions about the training set up. Would love to know any of the following if you are open to sharing:

1. How many epochs?
2. Do you have any loss curves for reference?
3. Wondering why you went with 8,4,2,1 for the codebooks instead of 4,2,1 - also what do you think of this?
4. You use 3 seconds as the default window (48000/16000) why not 0.8 seconds as in the snac paper - also what do you think of this?
5.  Why not at 24k why at 16k?
6. Do you have any suggestions for improving the model?

Thanks in advance for any information!

**Comment by hertz-pj** [comment]
Regarding questions 3, 4, and 5, please feel free to revise them as you see fit. I find the 0.8-second window to be excessively short, and I believe this was likely a concession to GPU memory limitations in the SNAC paper.
For question 2, you can refer to issue  #2 .
Additionally, our current research is more focused on single-codebook solutions.
